src/main/resources/biz_de.properties

```
@@ -933,8 +936,9 @@ Tenant.suffixDays = Tage
 TenantAware.invalidTenant = Fehler im Feld '${field}': Das Objekt muss zum gleichen Mandanten gehören.
 TenantController.cannotBecomeTenant = Sie können den gewählten Mandanten nicht verwalten.
 TenantController.cannotDeleteSelf = Sie können nicht Ihren eigenen Mandanten löschen.
-TenantController.select = Übernehmen
+TenantController.delete = Löschen...
```

Warum brauchen wir da überhaupt einen eigenen Key?

```suggestion
TenantController.delete = Löschen…
```